sw/device/silicon_creator/lib/shutdown_unittest.cc

```diff
@@ -537,6 +547,40 @@ TEST_F(ShutdownTest, RedactPolicyInvalid) {
   }
 }
 
+TEST_F(ShutdownTest, InitializeManufacturing) {
+  // OTP reads and alert setup should be skipped in the RAW and TEST_UNLOCKED
```

Comment:
nit: RAW should be removed from the comment.

Reply:
done in follow up